icevision/models/mmdet/utils.py

```diff
@@ -47,7 +47,7 @@ def param_groups(model):
         layers += [getattr(body, l) for l in body.res_layers]
 
     # add the neck module if it exists (DETR doesn't have a neck module)
-    layers += [model.neck for name in model.named_modules() if name == "neck"]
+    layers += [module for name, module in model.named_modules() if name == "neck"]
```

Comment:
This change was change merged in this [PR](https://github.com/airctic/icevision/pull/1029)